src/EventStore.Core/Services/Transport/Grpc/Streams.BatchAppend.cs

```diff
@@ -36,8 +36,11 @@ partial class Streams<TStreamId> {
 				requestStream, responseStream,
 				context.GetHttpContext().User, _maxAppendSize, _writeTimeout,
 				GetRequiresLeader(context.RequestHeaders));
-			
-			await worker.Work(context.CancellationToken);
+			try {
+				await worker.Work(context.CancellationToken);
+			} catch(Exception) {
```

Comment:
We should only trap the specific exception that's flooding the logs

Reply:
Done